docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/create-full-profile.md

```diff
@@ -34,7 +34,13 @@ Create a full profile by first adding infrastructure layers composed of an Opera
 
   To learn how to create multiple profile versions that use the same name, check out [Version a Cluster Profile](../modify-cluster-profiles/version-cluster-profile.md).
 
-5. Select the infrastructure provider or managed Kubernetes for your environment and click **Next**.
+5. Select the infrastructure provider, managed Kubernetes, or Technical Preview cloud type for your environment and click **Next**.
+
+  :::info
+
+  Cluster profiles created from cloud types that are under Technical Preview are intended for clusters that a cloud provider deploys using Palette's generic framework built upon the open-source Cluster API (CAPI) initiative. When creating this type of profile, you do not have to specify anything for the OS or Kubernetes layers. Out-of-the-box packs are provided for the network and storage profile layers. Additionally, Palette provides a cloud provider-specific storage pack. 
```

Comment:
I would remove that for 4.2. We only support Nutanix, not custom cloud support.

Reply:
We need to say something because users will see the Nutanix cloud type, so revised it to be Nutanix-specific: 

Nutanix is a Tech Preview cloud type intended for Nutanix clusters deployed using Palette's generic framework built upon the open-source Cluster API (CAPI) initiative.

When creating a Nutanix profile, you do not have to specify anything for the OS or Kubernetes layers. Palette provides out-of-the-box packs for the network and storage profile layers, including the **Nutanix CSI** storage pack.